src/python/build_management/build_manager.py

```diff
@@ -1154,13 +1154,18 @@ def _get_targets_list(bucket_path):
   """Get the target list for a given fuzz target bucket path. This is done by
   reading the targets.list file, which contains a list of the currently active
   fuzz targets."""
-  targets_list_path = os.path.join(
-      os.path.dirname(os.path.dirname(bucket_path)), TARGETS_LIST_FILENAME)
+  bucket_dir_path = os.path.dirname(os.path.dirname(bucket_path))
+  targets_list_path = os.path.join(bucket_dir_path, TARGETS_LIST_FILENAME)
   data = storage.read_data(targets_list_path)
   if not data:
     return None
 
-  return data.splitlines()
+  # Filter out targets which are not yet built.
+  targets = data.splitlines()
+  listed_targets = set(
+      os.path.basename(path.rstrip('/'))
```

Comment:
shouldn't list_blobs strip the / or is too problematic there ?

Reply:
I think it's good to leave the '/'. It's what `gsutil ls` returns.